.gitignore

```diff
@@ -143,4 +143,7 @@ examples/*/*.csv
 *.pkl
 
 # DB Files
-*.db
\ No newline at end of file
+*.db
+
+# Image Files
+*.png
```

Comment:
will this lead to png files in docs being ignored?

Reply:
oh that's a good point I'll change that